automation/bake/Dockerfile

```diff
@@ -11,20 +11,20 @@ RUN curl https://packages.cloud.google.com/apt/doc/apt-key.gpg | apt-key --keyri
 RUN apt-get update && apt-get install -y google-cloud-sdk
 
 ARG COMMIT_HASH=master
-ARG TESTNET_NAME=testworld
+ARG TESTNET_NAME=finalfinal2
 ARG CONFIG_FILE="/root/daemon.json"
 
-RUN curl https://raw.githubusercontent.com/MinaProtocol/mina/${COMMIT_HASH}/automation/terraform/testnets/${TESTNET_NAME}/genesis_ledger.json -o "${CONFIG_FILE}"
+#RUN curl https://raw.githubusercontent.com/MinaProtocol/mina/${COMMIT_HASH}/automation/terraform/testnets/${TESTNET_NAME}/genesis_ledger.json -o "${CONFIG_FILE}"
 
-RUN head ${CONFIG_FILE}
+#RUN head ${CONFIG_FILE}
 
 # Create the defautl config dir and an empty config
 RUN mkdir -p /root/.mina-config && echo "{}" > /root/.mina-config/daemon.json
 
-RUN mina daemon -config-file ${CONFIG_FILE} -generate-genesis-proof true; \
```

Comment:
I think this would remove the ability to have configs that aren't pre-baked into the genesis_ledger? (for example, for a future testnet?) I would vote we keep that functionality so keep this part of the bake script

Reply:
only production builds require baking as it is, as long as you set generateGenesisProof=true in the terraform, testnets can use images right from CI and the vast majority of networks are run that way.

I made this change for finalfinal2 so that the baked images were as close as possible to the original images in CI that have more testing, and so that using a baked docker image tests the default config file + entrypoint. I'm fine with un-commenting this before merging it back into master but I think this is a safer and more conservative way to bake images for production deployments/public networks.